.changeset/famous-feet-kneel.md

```diff
@@ -0,0 +1,9 @@
+---
+'lit': minor
+'lit-element': minor
+'lit-html': minor
```

Comment:
does lit-html need to be minor?

Reply:
It does not, thank you! Only `reactive-element` & `ts-transformers` have logic changes. `lit` and `lit-element` re-export the new decorator so I assume they should remain.

Thanks for catching this.